src/meta/src/hummock/manager/mod.rs

```diff
@@ -2557,6 +2612,13 @@ impl HummockManager {
                 }
             }
         }
+
+        tracing::info!(
```

Comment:
would this log be too frequently?  I think we do not need to print it every times

Reply:
change it to debug,  we can provide a separate interface to get table_vnode_partition if needed